### PR TITLE
Add Boston Release demo resources

### PR DESCRIPTION
**Pull request by hyork**
Adding Boston demo page and accompanying design system.

### CODE REVIEW

File: Releases/Boston/README.md [hunk 1]
```diff
@@ -0,0 +1,16 @@
+# Skuid Boston
+
+Resources to guide you through the features that come with Skuid Boston release (v13.X), including a demo page and an accompanying design system. 
+
+## About Skuid Boston
+Link to [Skuid Boston's Release Notes](https://docs.skuid.com/latest/en/release-notes.html).
+
+
+## Usage
+Once you have Skuid Boston installed in your sandbox or dev org, you can import the demo page XML and the Design System linked below.
```
Comment (itmightbecody) [inline Releases/Boston/README.md:10]: Capitalization things.

```suggestion
Once you have Skuid Boston installed in your sandbox or dev org, you can import the demo page XML and the design system linked below.
```

File: Releases/Boston/README.md [hunk 2]
```diff
@@ -0,0 +1,16 @@
+# Skuid Boston
+
+Resources to guide you through the features that come with Skuid Boston release (v13.X), including a demo page and an accompanying design system. 
+
+## About Skuid Boston
+Link to [Skuid Boston's Release Notes](https://docs.skuid.com/latest/en/release-notes.html).
```
Comment (itmightbecody) [inline Releases/Boston/README.md:6]: Let's link this to the version-specific release notes so this link stays accurate in the future.
The `v13.0.4` docs are publishing now, so this link should be live by around 3:40 PM EST

Also adding a line break between the header and the section text.

```suggestion

Link to [Skuid Boston's Release Notes](https://docs.skuid.com/v13.0.4/en/release-notes.html).
```

File: Releases/Boston/README.md [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+
+
+## Usage
+Once you have Skuid Boston installed in your sandbox or dev org, you can import the demo page XML and the Design System linked below.
+
+- API: v2
+- Design System: Download [BostonDemo Design System](BostonDemo.designsystem). In Skuid, go to Design Systems and import the downloaded design system.
+- Demo page: Download the [Demo Page](BostonDemo.xml). In Skuid, create a new page and select Import XML file to import this demo page.
```
Comment (itmightbecody) [inline Releases/Boston/README.md:14]: Capitalization
```suggestion
- Demo page: Download the [demo page](BostonDemo.xml). In Skuid, create a new page and select Import XML file to import this demo page.
```

File: Releases/Boston/README.md [hunk 4]
```diff
@@ -0,0 +1,16 @@
...
+
+## About Skuid Boston
+Link to [Skuid Boston's Release Notes](https://docs.skuid.com/latest/en/release-notes.html).
+
+
+## Usage
+Once you have Skuid Boston installed in your sandbox or dev org, you can import the demo page XML and the Design System linked below.
+
+- API: v2
+- Design System: Download [BostonDemo Design System](BostonDemo.designsystem). In Skuid, go to Design Systems and import the downloaded design system.
```
Comment (itmightbecody) [inline Releases/Boston/README.md:13]: Capitalization + a word
```suggestion
- Design System: Download the [BostonDemo design system](BostonDemo.designsystem). In Skuid, go to Design Systems and import the downloaded design system.
```

File: Releases/Boston/README.md [hunk 5]
```diff
@@ -0,0 +1,16 @@
+# Skuid Boston
+
+Resources to guide you through the features that come with Skuid Boston release (v13.X), including a demo page and an accompanying design system.
```
Comment (Rnhatch) [inline Releases/Boston/README.md:3]: Would it be better to use lower case "x". like:  Boston release (v13.x)

**Review by itmightbecody — CHANGES_REQUESTED** [review]
Only minor Readme changes. The page and design system themselves are excellent! 🙌

File: Releases/Boston/README.md (hunk 5 above)
Comment (itmightbecody) [reply Releases/Boston/README.md:3]: Aye, I'd lean toward a lowercase `x`, but either is fine.

**Review by Rnhatch — COMMENTED** [review]
@hyork  This design system does not seem to install in any Platform site I can get to.    Is this your experience too?

**Comment by hyork** [comment]
@Rnhatch I was able to import it to the demo.skuidsite.com (link [here](https://demo.skuidsite.com/ui/theme/compose/4cb80094-bfda-4f26-bcc8-91d439766be4))

**Comment by hyork** [comment]
Thanks @itmightbecody 🙌 ! I've committed your suggestions and also changed 13.X to 13.x. Ready for re-review.

**Review by itmightbecody — APPROVED** [review]
Hooray! 🎉  Thanks for the quick changes.

@Rnhatch @hyork I also tested the design system on a Platform site, and it worked for me. Not sure what the issue could be.

I'll mark this as approved on my end, but feel free to continue tweaking if you find an issue.

**Comment by Rnhatch** [comment]
Ok.  If you've gotten it to work in Platform site - I'll trust you.  I'll let @itmightbecody do actually approval.

**Pull request merged**